### PR TITLE
General improvements all over the app

### PR DESCRIPTION
**Pull request by brokelyn**
Please have mercy that this PR is all over the place. The git commit should be fairly sorted

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Create an admin account to log in. You can then import the `pb_schema.json` in S
 4. Create a local `.env` file with `cp .env.template .env`.
 5. Run with `npm run dev`.
 
+The pocketbase executable should be run in a separate folder.
```
Comment (nilsalm) [inline README.md:24]: 👍

File: src/lib/components/Item.svelte [hunk 2]
```diff
@@ -48,7 +43,7 @@
 				<input type="hidden" name="quantity" value={item.quantity - 1} />
 			</form>
 			<form action="?/increaseItem" method="POST" use:enhance>
-				<button class="text-gray-700 font-semibold px-2 border-2 border-primary rounded">
+				<button class="text-gray-700 font-semibold text- px-2 border-2 border-primary rounded">
```
Comment (nilsalm) [inline src/lib/components/Item.svelte:46]: is `text-` really a class? What does it do?

File: src/lib/components/List.svelte [hunk 3]
```diff
@@ -10,15 +8,17 @@
 
 <a
 	href={`/lists/${list.id}`}
-	class="rounded-lg aspect-square relative {list.isTemplate
+	class="hover:scale-103 rounded-lg aspect-square relative {list.isTemplate
```
Comment (nilsalm) [inline src/lib/components/List.svelte:11]: I like that!! Would you mind adding a transition to smooth it a bit?

File: src/routes/+layout.svelte [hunk 4]
```diff
@@ -2,45 +2,40 @@
...
-					<form
-						method="POST"
-						action="/logout"
-						use:enhance={() => {
-							return async ({ result }) => {
-								pb.authStore.clear();
-								await applyAction(result);
-							};
-						}}
+				<NavButton href="/lists" text="Lists" />
```
Comment (nilsalm) [inline src/routes/+layout.svelte:17]: very nice!

File: src/routes/lists/+page.svelte [hunk 5]
```diff
@@ -5,28 +5,42 @@
...
-					<List {list} />
-				{/each}
+			<div class="flex flex-col gap-4 w-full">
+				<div class="container m-auto grid grid-cols-2 md:grid-cols-4 gap-4">
+					{#each lists as list}
+						<List {list} />
+					{/each}
+				</div>
+				{#if templateLists.length > 0}
+					<Title title={'My Templates'} />
```
Comment (nilsalm) [inline src/routes/lists/+page.svelte:37]: I know this was like this before, but I think the prop could even be `title="My Templates"`, right?

File: src/lib/components/SubTitle.svelte [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+	export let title: string;
+</script>
+
+<div>
+	<div class="text-xl text-primary font-semibold">{title}</div>
+	<div class="h-1 my-2 bg-primary w-16" />
```
Comment (nilsalm) [inline src/lib/components/SubTitle.svelte:7]: Would you mind trying `bg-secondary` here? Might be cool, might not.

File: static/favicon.png
Comment (nilsalm) [inline static/favicon.png:1]: Fantastic, love it!! I have some ideas too for the app logo in the future, but this is a huge improvement already!

**Review by nilsalm — APPROVED** [review]
Thank you so much for you effort and contribution, it really warms my heart to see someone else say "yes, and..." and jump in on this project! ❤️

I left some small thoughts here and there but overall this is a good step forward and can definitely be merged after you had a look at the ideas.

File: src/lib/components/Item.svelte (hunk 2 above)
Comment (brokelyn) [reply src/lib/components/Item.svelte:46]: no its not. I am just stupid not removing stuff

File: src/lib/components/List.svelte (hunk 3 above)
Comment (brokelyn) [reply src/lib/components/List.svelte:11]: no sure can do

File: src/routes/lists/+page.svelte (hunk 5 above)
Comment (brokelyn) [reply src/routes/lists/+page.svelte:37]: yes would look better

File: src/lib/components/SubTitle.svelte (hunk 6 above)
Comment (brokelyn) [reply src/lib/components/SubTitle.svelte:7]: nope can do :D

Comment (brokelyn) [reply src/lib/components/SubTitle.svelte:7]: I tried it and the contrast is super bad when the background is white. So I would keep it `primary`

**Pull request merged**